src/Nowruz/modules/settings/components/orgTeam/orgTeam.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'src/components/_primitives.scss';
+
+.containerSector {
+  @apply w-full flex flex-col md:flex-row gap-6 md:gap-8;
```

Comment:
(it's just for me)
when we use @apply mixin, can't we use inline predefined class?

Reply:
this .containerSector class is used more than one time in the page, so I added a class to make possible changes easier